plugins/styleguide/assets/javascripts/discourse/components/sections/organisms/user-about.gjs

```diff
@@ -288,6 +288,11 @@ const UserAbout = <template>
               </span>&nbsp;{{i18n "user.staff_counters.deleted_posts"}}
             </a>
           </div>
+          <div>
+            <span class="silencings">
+              {{@dummy.user.num_of_silencings}}
+            </span>&nbsp;{{i18n "user.staff_counters.silencings"}}
```

Comment:
I think that CSS rule `margin-right` would be better than `&nbsp;`

Reply:
<img width="1946" height="1234" alt="image" src="https://github.com/user-attachments/assets/243278ed-bdd3-4f13-8268-0de2320383ec" />

Actually I checked the styleguide and found that user-about is completely broken here, maybe we should fix it in another PR instead of here?